test/scripts/e2e-cp.sh

```diff
@@ -190,6 +190,8 @@ FOCUS=$(echo ${@:1} | sed 's/ /\\s/g')
 pushd e2e
 
 go mod download
+sudo df -h
+sudo du -a / --exclude="/proc" --exclude="/sys" --exclude="/dev" | sort -rn | head -20  2>/dev/null
```

Comment:
@jluhrsen I picked https://github.com/ovn-org/ovn-kubernetes/actions/runs/8821621318/job/24218898743?pr=4305 which is a control plane lane and saw:

```
+ sudo du -a / --exclude=/proc --exclude=/sys --exclude=/dev
+ sort -rn
+ head -20
63052754	/
sort: write failed: 'standard output': Broken pipe
21725260	/var
sort: write error
21548828	/var/lib
21271808	/usr
20675084	/var/lib/docker
11899068	/var/lib/docker/overlay2
11453876	/usr/local
9724584	/opt
8806476	/opt/hostedtoolcache
8761456	/var/lib/docker/volumes
5520236	/usr/local/.ghcup
5464364	/usr/local/.ghcup/ghc
5234104	/opt/hostedtoolcache/CodeQL
5234100	/opt/hostedtoolcache/CodeQL/2.17.0
5234096	/opt/hostedtoolcache/CodeQL/2.17.0/x64
5234092	/opt/hostedtoolcache/CodeQL/2.17.0/x64/codeql
4404[164](https://github.com/ovn-org/ovn-kubernetes/actions/runs/8821621318/job/24218898743?pr=4305#step:10:165)	/home
4194332	/mnt
4194308	/mnt/swapfile
4086324	/usr/share
```
something to do with rendering in the sort command which we want to fix?

+ do you wanna add the mem output in a human readable form ? like mb, gb etc? 4086324 is hard haha just a suggestion

Reply:
updated to fix it. my 2>/dev/null was in the wrong spot

Reply:
sort ends up being wrong when you use -h for human readable since something like 2M is sorted higher than 1G which is wrong in our case here.

Reply:
> sort ends up being wrong when you use -h for human readable since something like 2M is sorted higher than 1G which is wrong in our case here.

ah I see thanks for explaining this! makes sense